library/src/main/java/com/schibsted/spain/barista/assertion/BaristaVisibilityAssertions.kt

```diff
@@ -36,18 +36,23 @@ object BaristaVisibilityAssertions {
         onView(withId(resId)).check(matches(withText(text)))
     }
 
+    /**
```

Comment:
I hope in the next refactor this comment is not necessary anymore... 🙏

Reply:
It's not because we'll move the code somewhere else. But I was thinking about leaving some similar documentation. What problem do you see with the comment? Don't you think it's a good thing to document the not-so-obvious behaviors of our methods?

Reply:
Having documentation comments is actually a good thing, IMO, but they aren't really all that useful in a private method. It would make more sense to comment the interface the users will actually work with.

Reply:
Yes it is but I was talking about destroying this so ugly code and make it beauty, so so beauty that you won't need this documentation 👍

Reply:
Good luck with that 🤣 
Let's talk about it in the refactor PR, maybe we can achieve it.